test/fixtures/es-module-specifiers/dir-with-main/main.js

```diff
@@ -0,0 +1 @@
+module.exports = 'main';
```

Comment:
Most of the files we have are terminated by newlines. Any reason why this one isn't?
```suggestion
module.exports = 'main';

```
I hope this doesn't end up adding a visible blank line. :crossed_fingers: